addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -758,6 +758,9 @@ var SnippetEditor = Widget.extend({
             if (!val.selector.is(this.$target)) {
                 return;
             }
+            if (val.data.string) {
+                $optionsSection[0].querySelector('we-title > span').textContent = val.data.string;
+            }
```

Comment:
@bvr-odoo Your commit message talk about the "string" attribute but you seem to use the data-string (which is not translated)

Reply:
@qsm-odoo  I use a `string` attribute (not data-string)
```xml
<div data-selector="main:has(.o_appointment_index)" data-page-options="True" data-no-check="true" string="Appointments Page">
```
It works because of this line in `_computeSnippetTemplates` :
```js
'data': _.extend({string: $style.attr('string')}, $style.data()),
```